addon/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+// please write your own index.js that exports legit reducers
+//
+// import { combineReducers } from 'redux';
+// const foobar = (state, action) => {
+//   //reducer code here
+// };
+//
+// export default combineReducers({
+//   foobar
+// })
+import Ember from 'ember';
+
+export default function(combine) {
```

Comment:
I'm still a bit confused with what exactly this is exporting here, so I'm sure I'm missing something. Should this not just be a no-op reducer, e.g.:
```
export default function(state) {
  Ember.warn(...);
  return state;
});
```
otherwise, lgtm